packages/integrations/components/integrationsManager/index.js

```diff
@@ -95,48 +85,10 @@ const IntegrationsManager = ({
 
 IntegrationsManager.defaultProps = {
   integrations: {},
-  hydratedComponents: [],
 };
 
 IntegrationsManager.propTypes = {
-  integrations: PropTypes.shape({
-    googleAnalytics: PropTypes.shape({
-      trackingId: PropTypes.string,
-    }),
-  }),
-  isHydrated: PropTypes.bool.isRequired,
```

Comment:
So this will just remove all the caching logic entirely? I thought we discussed the possibility of bypassing it only for certain components. I'm not opposed to this approach if it works, though maybe worth revisiting later.

Reply:
I definitely think we should revisit later, but the current caching mechanism seem to not be working at all, so this just removes all of that logic entirely.